app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/activity/BlockedUsers.java

```diff
@@ -0,0 +1,38 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.activity;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class BlockedUsers extends AppCompatActivity {
```

Comment:
Could you please update naming convention for activities ie. BlockedUsersActivity